CMSgov/mint-app
# Easi 3567/beneficiaries updates

# EASI-3567

## Changes and Description

- Add the enum underserved to the participants type enum list.
   - FE translations and work 
- Add disease specific group question to plan beneficiaries.
- FE addition of the question and hooking.

- FE Translation renaming of medicaid to MEDICAID: `Medicaid/Children's Health Insurance Program (CHIP)`


## How to test this change

1. Bring up the application 
2. Navigate to the beneficiaries page
3. Confirm the options are added / updated correctly
  a. Underserved
  b. Medicaid/Children's Health Insurance Program (CHIP)
4. Select Disease Specific for a beneficiary to make the new disease specific group question appear
5. Fill out the diseases specific group question.

6. Navigate back to the page to make sure the data saved as expected.
7. View the data in read only view as well to ensure it is as expected


If desired, the new fields can also be tested with the updated postman collection.



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
